.github/workflows/cclf-import-apply.yml

```diff
@@ -0,0 +1,44 @@
+name: cclf-import apply terraform
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/cclf-import-apply.yml
+      - terraform/modules/bucket/**
+      - terraform/modules/key/**
+      - terraform/modules/function/**
+      - terraform/modules/queue/**
+      - terraform/modules/subnets/**
+      - terraform/modules/vpc/**
+      - terraform/services/cclf-import/**
+  workflow_dispatch: # Allow manual trigger
+
+jobs:
+  terraform-apply:
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./terraform/services/cclf-import
+    strategy:
+      fail-fast: false
+      matrix:
+        app: [bcda]
+        env: [dev, test, prod]
+    env:
+      TF_VAR_app: ${{ matrix.app }}
+      TF_VAR_env: ${{ matrix.env }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./actions/setup-tfenv-terraform
+      - uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::${{ matrix.app == 'bcda' && secrets[format('{0}_{1}_ACCOUNT', matrix.app, matrix.env)] || secrets.BCDA_ACCOUNT }}:role/delegatedadmin/developer/${{ matrix.app }}-${{ matrix.env }}-github-actions
+          aws-region: ${{ vars.AWS_REGION }}
+      - run: terraform init -reconfigure -backend-config=../../backends/$TF_VAR_app-$TF_VAR_env.s3.tfbackend
+      - run: terraform apply -auto-approve
+
```

Comment:
Please add a cclf-import-plan workflow as well.

Reply:
Once the plan workflow is added we can use that to see the terraform plan output for verification.

Reply:
Plan workflow added. Is running the plan workflow something that should happen locally or during another type of check?

Reply:
idk about locally but the plan should trigger sucessfully before the apply triggers for sure.

Reply:
Should be good to go now.